packages/components/src/components/contract-card/contract-card-items/__tests__/multiplier-card-body.spec.tsx

```diff
@@ -96,7 +96,6 @@ describe('MultiplierCardBody', () => {
         // @ts-expect-error Check if error is gone after migrating MultiplierCardBody to TS
         render(<MultiplierCardBody {...mock_props} />);
 
-        expect(screen.getByText(mock_props.getCardLabels().NOT_AVAILABLE)).toBeInTheDocument();
```

Comment:
I think you need to update the description for this test case as well.

Reply:
Updated, thanks